src/mem/snpsecrets.rs

```diff
@@ -58,3 +58,17 @@ impl SnpSecrets {
     funcs!(svsm_guest_vmpl, u8);
     funcs!(vmpck0, [u8; VMPCK_SIZE]);
 }
+
+pub fn disable_vmpck0() {
+    let svsm_secrets_va: VirtAddr = get_svsm_secrets_page();
+    let svsm_secrets_ptr: *mut SnpSecrets = svsm_secrets_va.as_mut_ptr();
```

Comment:
I saw this in a previous patch, but can we simplify this by doing:

let svsm_secrets: *mut SnpSecrets = get_svsm_secrets_page().as_mut_ptr();

? (and below)